### PR TITLE
LICENSE.html file: update the script which creates the LICENSE.html file

### PR DESCRIPTION
**Pull request by rgetz**
- fix bug #26, follow 302 redirects, and break on errors
 - make things html compliant by adding a few <p></p> tags
 - remove language files which are not there
 - update mailing address, since we moved

Signed-off-by: Robin Getz <robin.getz@analog.com>

### CODE REVIEW

File: scripts/legal_info_html.sh [hunk 1]
```diff
@@ -258,22 +254,21 @@ convert_md2html LICENSE.md >> ${FILE}
...
-Norwood, MA.
-02062
+1 Analog Way
+Wilmington, Massachusetts
+01887
 USA
 </pre>
-
-Please write “<i>source for the ${TARGET}</i>” in the memo line of your payment.
+<p>Please write “<i>source for the ${TARGET}</i>” in the memo line of your payment.
```
Comment (commodo) [inline scripts/legal_info_html.sh]: is this missing an equivalent `</p>` tag ?

**Comment by mhennerich** [comment]
Hmm - running the page by an HTML validator shows this and also many other issues:
https://jsonformatter.org/html-validator

Error on->  Line :12 Column :8
Message :Opening and ending tag mismatch: link line 11 and head

Error on->  Line :50 Column :12
Message :AttValue: " or ' expected

Error on->  Line :50 Column :12
Message :attributes construct error

Error on->  Line :50 Column :12
Message :Couldn't find end of Start Tag div line 50

Error on->  Line :68 Column :7
Message :Opening and ending tag mismatch: p line 65 and div

Error on->  Line :12896 Column :8
Message :Opening and ending tag mismatch: hr line 12895 and body

Error on->  Line :12897 Column :9
Message :Opening and ending tag mismatch: hr line 12828 and html

Error on->  Line :12898 Column :1
Message :Premature end of data in tag hr line 12478

Error on->  Line :12898 Column :1
Message :Premature end of data in tag hr line 11946

Error on->  Line :12898 Column :1
Message :Premature end of data in tag hr line 11595

Error on->  Line :12898 Column :1
Message :Premature end of data in tag hr line 7282

Error on->  Line :12898 Column :1
Message :Premature end of data in tag hr line 7156

Error on->  Line :12898 Column :1
Message :Premature end of data in tag hr line 7122

Error on->  Line :12898 Column :1
Message :Premature end of data in tag hr line 6607

Error on->  Line :12898 Column :1
Message :Premature end of data in tag hr line 6431

Error on->  Line :12898 Column :1
Message :Premature end of data in tag hr line 5918

Error on->  Line :12898 Column :1
Message :Premature end of data in tag hr line 5405

Error on->  Line :12898 Column :1
Message :Premature end of data in tag hr line 4938

Error on->  Line :12898 Column :1
Message :Premature end of data in tag hr line 4417

Error on->  Line :12898 Column :1
Message :Premature end of data in tag hr line 3891

Error on->  Line :12898 Column :1
Message :Premature end of data in tag hr line 3375

Error on->  Line :12898 Column :1
Message :Premature end of data in tag hr line 3348

Error on->  Line :12898 Column :1
Message :Premature end of data in tag hr line 3304

Error on->  Line :12898 Column :1
Message :Premature end of data in tag hr line 2953

Error on->  Line :12898 Column :1
Message :Premature end of data in tag hr line 2921

Error on->  Line :12898 Column :1
Message :Premature end of data in tag hr line 2769

Error on->  Line :12898 Column :1
Message :Premature end of data in tag hr line 2206

Error on->  Line :12898 Column :1
Message :Premature end of data in tag hr line 1847

Error on->  Line :12898 Column :1
Message :Premature end of data in tag hr line 1326

Error on->  Line :12898 Column :1
Message :Premature end of data in tag hr line 976

Error on->  Line :12898 Column :1
Message :Premature end of data in tag hr line 626

Error on->  Line :12898 Column :1
Message :Premature end of data in tag hr line 259

Error on->  Line :12898 Column :1
Message :Premature end of data in tag p line 49

Error on->  Line :12898 Column :1
Message :Premature end of data in tag p line 46

Error on->  Line :12898 Column :1
Message :Premature end of data in tag p line 37

Error on->  Line :12898 Column :1
Message :Premature end of data in tag p line 35

Error on->  Line :12898 Column :1
Message :Premature end of data in tag p line 29

Error on->  Line :12898 Column :1
Message :Premature end of data in tag hr line 25

Error on->  Line :12898 Column :1
Message :Premature end of data in tag body line 13

Error on->  Line :12898 Column :1
Message :Premature end of data in tag link line 10

Error on->  Line :12898 Column :1
Message :Premature end of data in tag link line 9

Error on->  Line :12898 Column :1
Message :Premature end of data in tag link line 8

Error on->  Line :12898 Column :1
Message :Premature end of data in tag meta line 6

Error on->  Line :12898 Column :1
Message :Premature end of data in tag meta line 5

Error on->  Line :12898 Column :1
Message :Premature end of data in tag meta line 4

Error on->  Line :12898 Column :1
Message :Premature end of data in tag head line 3

Error on->  Line :12898 Column :1
Message :Premature end of data in tag html line 2

**Comment by rgetz** [comment]
I ran it on:
https://validator.w3.org/nu/#file

Document checking completed. No errors or warnings to show.

Used the HTML parser.
Total execution time 858 milliseconds.

**Comment by rgetz** [comment]
Also ran it on : https://www.freeformatter.com/html-validator.html 
The document is valid and conforms to best practices and standards. Good job!

The ones that throw errors (I think) are false positives, things like html's link & meta tags, don't have end tags:
https://www.w3schools.com/tags/tag_link.asp
https://www.w3schools.com/tags/tag_meta.asp

This is more of a xml thing, that the other validators are checking.

I fixed the missing </p> tag that @commodo pointed out, and a few other nitpik things that some of the other services point out, but I think it is OK now.

**Pull request merged**